src/components/forms/RegisterForm.js

```diff
@@ -67,10 +68,10 @@ class RegisterForm extends Component {
           <input type="submit" value="Register" />
         </form>
         <h2>All ready have an account with Skychute?</h2>
-        {/* <a href="/signin">Sign In</a> */}
+        <a href="/">Sign In</a>
```

Comment:
This one is also fixed. I'll style it as a button later. I'll try to style both the register form & sign in form today.